api/models/ormconfig.ts

```diff
@@ -0,0 +1,22 @@
+import { ConnectionOptions } from 'typeorm';
+const ORMConfig: ConnectionOptions = {
+  type: 'postgres',
+  host: process.env.DB_HOST,
+  port: parseInt(process.env.DB_PORT),
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+  entities: [
+    __dirname + '/entity/*{.ts,.js}',
+  ],
+  migrations: [
+    __dirname + '/migrations/*{.ts,.js}',
+  ],
+  synchronize: process.env.NODE_ENV === 'production' ? false : true,
```

Comment:
This was formerly true for all environments. Is there any implications we need to be aware of from this change?

Reply:
According to[ the documentation](https://typeorm.io/#/connection-options):
>synchronize - Indicates if database schema should be auto created on every application launch. Be careful with this option and don't use this in production - otherwise you can lose production data. This option is useful during debug and development. As an alternative to it, you can use CLI and run schema:sync command. Note that for MongoDB database it does not create schema, because MongoDB is schemaless. Instead, it syncs just by creating indices.

I think now is a pretty good type to make the switch, but during development / testing I think having it be `true` is ideal, hence the env aware functionality.